modules/ROOT/pages/constraints/syntax.adoc

```diff
@@ -272,14 +362,28 @@ _This feature was introduced in Neo4j 5.3._
 |[PROPERTY] EXIST[ENCE]
 | Returns all property existence constraints, for both nodes and relationships.
 
+|NODE PROPERTY TYPE
+| Returns the node property type constraints.
+_This feature was introduced in Neo4j 5.9._
```

Comment:
In a table, use a label: 

```suggestion
label:new[Introduced in 5.9]
```

Reply:
this applies to more places than this one

Reply:
Yes, I did not want to comment for each though :)